src/formats/PDB.cpp

```diff
@@ -403,3 +423,9 @@ void check_values_size(const Vector3D& values, unsigned width, const std::string
         );
     }
 }
+
+PDBFormat::~PDBFormat() {
+  if (written_) {
```

Comment:
Could you please use 4 spaces for indentation?

Reply:
done.